test_runner/regress/test_tenant_conf.py

```diff
@@ -158,3 +166,53 @@ def test_tenant_config(neon_env_builder: NeonEnvBuilder):
                     "pitr_interval": 60,
                 }.items()
             )
+
+
+def test_creating_tenant_conf_after_attach(neon_env_builder: NeonEnvBuilder):
+    neon_env_builder.enable_remote_storage(
+        remote_storage_kind=RemoteStorageKind.LOCAL_FS,
+        test_name="test_creating_tenant_conf_after_attach",
+    )
+
+    env = neon_env_builder.init_start()
+    assert isinstance(env.remote_storage, LocalFsStorage)
+
+    # tenant is created with defaults, as in without config file
+    (tenant_id, timeline_id) = env.neon_cli.create_tenant()
+    config_path = env.repo_dir / "tenants" / str(tenant_id) / "config"
+    assert config_path.exists(), "config file is always initially created"
+
+    env.pageserver.stop()
+
+    remote_dir = (
+        env.remote_storage.root / "tenants" / str(tenant_id) / "timelines" / str(timeline_id)
+    )
+    found_any = False
+    for path in Path.iterdir(remote_dir):
+        if not path.is_file():
+            continue
+        found_any = True
+        break
+
+    assert found_any, "should had have files under timeline directory"
```

Comment:
What do we check, that there's an attach possible afterwards?
We could be more rigorous and check that all data is actually uploaded: https://github.com/neondatabase/neon/blob/0a095894035b4a1ebf660ad04fadbe77d5185475/test_runner/fixtures/neon_fixtures.py#L3260

```suggestion
    assert found_any, "should had have files under timeline remote storage directory"
```

Reply:
`wait_for_upload` was used there already.